target/linux/ar71xx/image/tp-link.mk

```diff
@@ -121,7 +121,7 @@ endef
 define Device/archer-cxx
   KERNEL := kernel-bin | patch-cmdline | lzma | uImageArcher lzma
   IMAGES := sysupgrade.bin factory.bin
-  IMAGE/sysupgrade.bin := append-rootfs | tplink-safeloader sysupgrade
+  IMAGE/sysupgrade.bin := append-rootfs | tplink-safeloader sysupgrade | append-metadata
```

Comment:
`tplink-safeloader` checks if kernel and rootfs fit in defined partitions in image, but as you use `append-metadata` here as a last build step and appended metadata is not stripped from sysupgrade image before writing it to flash, `check-size $$$$(IMAGE_SIZE)` is needed here.